src/Uno.UWP/Storage/ApplicationDataContainer.wasm.cs

```diff
@@ -20,165 +24,84 @@ public partial class ApplicationDataContainer
 
 		private class FilePropertySet : IPropertySet
 		{
-			private const string UWPFileName = ".UWPAppSettings";
-			private readonly Dictionary<string, string> _values = new Dictionary<string, string>();
-			private readonly string _folderPath;
-			private readonly string _filePath;
+			private readonly ApplicationDataLocality _locality;
 
 			public FilePropertySet(ApplicationData owner, ApplicationDataLocality locality)
 			{
-				StorageFolder folder;
-				switch (locality)
-				{
-					case ApplicationDataLocality.Local:
-						folder = owner.LocalFolder;
-						break;
-
-					case ApplicationDataLocality.Roaming:
-						folder = owner.RoamingFolder;
-						break;
-					case ApplicationDataLocality.LocalCache:
-						folder = owner.LocalCacheFolder;
-						break;
-
-					case ApplicationDataLocality.Temporary:
-						folder = owner.TemporaryFolder;
-						break;
-
-					default:
-						throw new ArgumentOutOfRangeException(nameof(locality));
```

Comment:
Is it creating a breaking change for data of published applications?

Reply:
@dr1rrb yes, it will create a breaking change, but considering it was unreliable it was probably not much used. But we can keep the existing reader to migrate the existing settings to the new format, I'll make the adjustment.